frontend/public/components/monitoring/receiver-forms/alert-manager-receiver-forms.tsx

```diff
@@ -371,16 +382,25 @@ const ReceiverBaseForm: React.FC<ReceiverBaseFormProps> = ({
       },
     );
   };
+  const { t } = useTranslation();
+  // t('alert-manager-receiver-form~PagerDuty')
+  // t('alert-manager-receiver-form~Webhook')
+  // t('alert-manager-receiver-form~Email')
+  // t('alert-manager-receiver-form~Slack')
+  const receiverTypeLabel = t('alert-manager-receiver-form~{{receiverTypeLabel}}', {
+    receiverTypeLabel: receiverTypes[formValues.receiverType],
+  });
 
   return (
     <div className="co-m-pane__body co-m-pane__form">
       <Helmet>
-        <title>{`${titleVerb} Receiver`}</title>
+        <title>{`${titleVerb} ${t('alert-manager-receiver-form~Receiver')}`}</title>
       </Helmet>
       <form className="co-m-pane__body-group" onSubmit={save}>
         <h1 className="co-m-pane__heading">
-          {titleVerb} {receiverTypes[formValues.receiverType]} {isDefaultReceiver && 'Default'}{' '}
-          Receiver
+          {titleVerb} {receiverTypeLabel}
+          {isDefaultReceiver && t('alert-manager-receiver-form~Default')}{' '}
```

Comment:
Same comment here. You probably want a "Default Receiver" message with the full phrase.